### PR TITLE
Enable full control of conjurations

### PR DESCRIPTION
**Pull request by PhilPJL**
Enable full control of conjurations from spells

- conjure animals
- conjure minor elementals
- conjure elemental

### CODE REVIEW

File: SolastaCommunityExpansion/Viewers/Displays/RulesDisplay.cs [hunk 1]
```diff
@@ -35,6 +35,13 @@ internal static void DisplayRules()
                 }
             }
 
+            toggle = Main.Settings.FullyControlAlliedConjurations;
```
Comment (ThyWoof) [inline SolastaCommunityExpansion/Viewers/Displays/RulesDisplay.cs:38]: is this SRD or should be under house rules? if SRD please move the UI.Label(""); above this setting to keep the first 3 in their own block.

File: SolastaCommunityExpansion/Models/ConjurationsContext.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+            // Conjure woodland beings (4) - not implemented
+
+            // Conjure elemental (5)
+            Air_Elemental.SetFullyControlledWhenAllied(controlled); // CR 5
+            Fire_Elemental.SetFullyControlledWhenAllied(controlled); // CR 5
+            Earth_Elemental.SetFullyControlledWhenAllied(controlled); // CR 5
+
+            // Conjure fey (6) - should all be CR 6 - not implemented in base game yet
+            //FeyBear.SetFullyControlledWhenAllied(controlled); // CR 4
```
Comment (ThyWoof) [inline SolastaCommunityExpansion/Models/ConjurationsContext.cs:33]: any collaterals to enable them now?

**Comment by ChrisPJohn** [comment]
Generally looks good, but waiting to approve until ThyWoof's comments are addressed.

File: SolastaCommunityExpansion/Viewers/Displays/RulesDisplay.cs (hunk 1 above)
Comment (PhilPJL) [reply SolastaCommunityExpansion/Viewers/Displays/RulesDisplay.cs:40]: I would say it's SRD.

File: SolastaCommunityExpansion/Models/ConjurationsContext.cs (hunk 2 above)
Comment (PhilPJL) [reply SolastaCommunityExpansion/Models/ConjurationsContext.cs:33]: Since this spell isn't fully implemented yet I would like to wait before addng.

**Review by ThyWoof — APPROVED** [review]

**Pull request merged**